store/storetest/channel_member_history_store.go

```diff
@@ -207,11 +207,14 @@ func testGetUsersInChannelAtChannelMembers(t *testing.T, ss store.Store) {
 	// available in the ChannelMemberHistory table. Instead, we'll fall back to the ChannelMembers table for a rough approximation
 	joinTime := int64(1000)
 	leaveTime := joinTime + 5000
-	store.Must(ss.Channel().SaveMember(&model.ChannelMember{
+	_, err = ss.Channel().SaveMember(&model.ChannelMember{
 		ChannelId:   channel.Id,
 		UserId:      user.Id,
 		NotifyProps: model.GetDefaultChannelNotifyProps(),
-	}))
+	})
+	if err != nil {
```

Comment:
You can replace it with `require.Nil(t, err)` which is more compact and explicit.